merkletree/pad.go

```diff
@@ -114,8 +114,9 @@ func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 func (pad *PAD) TB(key string, value []byte) (*TemporaryBinding, error) {
 	//FIXME: compute private index twice
 	//it would be refactored after merging VRF integration branch
-	index, _ := pad.computePrivateIndex(key)
-	tb := pad.currentSTR.sig
+	cStr := pad.currentSTR
```

Comment:
> But do we still need the policies field in STR then (for serializing, or sth else)?

I think we do.  Anything that gets signed needs to remain so that it can be verified later.  But also, in this case, the when you do the lookup from a previous epoch, you still need to use the key from the old policy.

So, I should have mentioned, `computePrivateIndex` should probably stay on the `pad` but we'll need a `computePrivateIndexInEpoch` as well.  Or, something like that.  You get the idea.